src/middlewared/middlewared/plugins/disk_/wipe.py

```diff
@@ -38,20 +38,17 @@ def _wipe(self, data):
                 for i in range(_32):
                     # wipe first 32MB
                     os.write(f.fileno(), to_write)
-                    os.fsync(f.fileno())
```

Comment:
I have following queries:
1) Why did we used to do this before ( my concern is introducing a regression by not doing it anymore ) ?
2) On freebsd `fsync(2)` does seem to make sure that this is propagated to disk ( https://www.freebsd.org/cgi/man.cgi?query=fsync&sektion=2&format=html )
3) What performance increase are we looking at ?

Reply:
1. if you look at history of this file you'll see that I rewrote this method from scratch so we were never doing `os.fsync` before. It was something I added when I rewrote it.
2. `fsync` to a character device is not propagated to disk the way you think it is on freeBSD
3. potentially many hours

Reply:
I am confused, `fsync` to a character device - which one is that in our context 😅  ?

That is indeed a lot of time, so we never needed to use `fsync` ?

Reply:
Right, we never needed fsync calls